cypress/fixtures/specific_article.json

```diff
@@ -0,0 +1,12 @@
+{
+  "articles": [
```

Comment:
The response it's not wrapped by "articles" as we sent you guys before in slack.

![image](https://user-images.githubusercontent.com/59778082/77226749-fd6a3680-6b7a-11ea-9e1f-0093c860c803.png)